Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -1342,8 +1342,12 @@ define([
             // selection depth to the stencil buffer to prevent ancestor tiles from drawing on top
             derivedCommand = DrawCommand.shallowClone(command);
             var rs = clone(derivedCommand.renderState, true);
+            // Stencil test is masked to the most significant 4 bits so the reference is shifted.
```

Comment:
For my knowledge, remind me what limitation the 4 bits creates, e.g., can't have more than 16 tilesets in the same pixel?

Reply:
See this comment when traversing 3D Tiles with the skip LOD optimization: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Cesium3DTilesetTraversal.js#L167

Instead of a depth of 255, the depth would be limited to 15.

Reply:
An unlikely, but potentially possible case now, right?  But this is limited to just classification?

At the least, please add appropriate comments if you haven't already in case we need to investigate this in the future.

Reply:
It's still very unlikely the selection depth will reach 15. I'll add a comment.